### PR TITLE
Frontend logging: Remove Sentry javascript agent support

### PR DESCRIPTION
**Pull request by domasx2**
**What is this feature?**

Faro has superseded Sentry for monitoring Grafana frontend. Sentry was marked as deprecated with https://github.com/grafana/grafana/pull/60165 and is to be removed in v10. This PR removes Sentry support from the codebase, leaving Faro as the only default option.

# Release notice breaking change

The deprecated support for monitoring Grafana frontend using Sentry javascript agent has been removed in this release. If you have frontend logging enabled and are sending telemetry to Sentry by setting `sentry_dsn` configuration property, this will no longer work. Otherwise, if frontend logging is enabled, it will now automatically use Grafana Faro agent.

### CODE REVIEW

**Review by ashharrison90 — APPROVED** [review]
great job removing this 👏 

might be worth adding slightly more to the breaking change notice to let people know what they can do (e.g. switch to Faro!)

**Comment by domasx2** [comment]
> great job removing this clap
> 
> might be worth adding slightly more to the breaking change notice to let people know what they can do (e.g. switch to Faro!)

Thanks, expanded the breaking change notice a bit!

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 1]
```diff
@@ -1280,31 +1280,19 @@ Syslog tag. By default, the process's `argv[0]` is used.
...
+Faro javascript agent is initialized. Default is `false`.
 
 ### custom_endpoint
 
-Custom HTTP endpoint to send events captured by the Sentry agent to. Default, `/log`, will log the events to stdout.
-
-### sample_rate
-
-Rate of events to be reported between `0` (none) and `1` (all, default), float.
+Custom HTTP endpoint to send events captured by the Faro agent to. Default, `/log-grafana-javascript-agent`, will log the events to stdout.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-grafana/_index.md:1287]:
```suggestion
Custom HTTP endpoint to which events captured by the Faro agent are sent. Default, `/log-grafana-javascript-agent`, will log the events to stdout.
```

**Review by chri2547 — COMMENTED** [review]
One suggestion, docs approved. Thanks for the contribution!

File: public/app/core/services/echo/EchoSrv.ts [hunk 2]
```diff
@@ -14,6 +13,5 @@ export const reportPerformance = (metric: string, value: number) => {
   });
 };
 
-// Sentry will process the error, adding its own metadata, applying any sampling rules,
-// then push it to EchoSrv as SentryEvent
-export const reportError = (error: Error) => captureException(error);
+// Farp will process the error, then push it to EchoSrv as GrafanaJavascriptAgent event
```
Comment (leventebalogh) [inline public/app/core/services/echo/EchoSrv.ts:16]: Typo
```suggestion
// Faro will process the error, then push it to EchoSrv as GrafanaJavascriptAgent event
```

**Pull request merged**